<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Selectscene</class>
 <widget class="QWidget" name="Selectscene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/resource/select.png</pixmap>
   </property>
  </widget>
  <widget class="QComboBox" name="playerBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="rivalBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="randomBotton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>随机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>华农不死人</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
